### PR TITLE
Audit article: InfoDev-2682

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e91ad9f217/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b2e75bfceb/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson For your review please.

**Comment by cyrichardson** [comment]
@RackerKate On it.

**Comment by cyrichardson** [comment]
@RackerKate Just a couple of suggestions here: 
[migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2554311/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0f48bfd899/how-to/).

File: content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md [hunk 1]
```diff
@@ -158,168 +177,195 @@ Ubuntu 12.0.4 LTS includes Python 2.7.2. If you need a different version or an a
...
         sudo /sbin/chkconfig --level 234 tomcat on
         sudo /sbin/chkconfig --list tomcat
 
-    The expected output looks as follows:
+     The output should be similar to the following example:
 
         tomcat 0:off 1:off 2:on 3:on 4:on 5:off 6:off
 
-14. Because Tomcat is running on port 8080, ensure iptables will not interfere with connectivity.
+14. Because Tomcat is running on port 8080, ensure that iptables doesn't
```
Comment (catlook) [inline content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md]: is iptables singular or plural? maybe "iptables don't interfere"?

File: content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md [hunk 2]
```diff
@@ -158,168 +177,195 @@ Ubuntu 12.0.4 LTS includes Python 2.7.2. If you need a different version or an a
...
+    interfere with connectivity.
 
-    To learn about iptables, see [Introduction to iptables](/how-to/introduction-to-iptables).
+     To learn about iptables, see [Introduction
+     to iptables](/how-to/introduction-to-iptables).
 
-15. Test Tomcat by looking up the cloud server IP from the Rackspace Cloud Control Panel and opening the URL in a browser (for example, `http://<ip_address>:8080/`).
+15. Test Tomcat by looking up the Internet Protocol (IP) address for the cloud
```
Comment (catlook) [inline content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md]: Cloud Server (use caps for the product)

Comment (RackerKate) [reply content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md]: Catherine says "Cloud Servers is the product and cloud servers when generic use."

Comment (catlook) [reply content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md]: that surprises me.  she told me always cap the product.  this is referring to a rackspace Cloud Server....not a random cloud server from another provider.

Comment (RackerKate) [reply content/cloud-servers/migrating-an-application-based-on-backbonejs-nodejs-and-mongodb-from-amazon-web-services.md]: @cyrichardson What say you?

**Pull request merged**